api/src/Responder/Responder/BaseResponder.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 16.07.15
+ * Time: 1:08
+ */
+
+namespace Spira\Responder\Responder;
+
+use Spira\Responder\Contract\ResponderInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class BaseResponder implements ResponderInterface
+{
+    /**
+     * Return an error response.
+     *
+     * @param string $message
+     * @param int $statusCode
+     *
+     * @throws HttpException
+     *
+     */
+    public function error($message, $statusCode)
```

Comment:
This has been discussed - we intend to keep these error statuses so the exception handler can use them. Controllers should not call them however.